### PR TITLE
login/register issues

### PR DESCRIPTION
**Pull request by xshi0603**
WIP: dont approve yet

EDIT: done

### CODE REVIEW

**Comment by xshi0603** [comment]
fixes #35 
fixes #31

**Comment by xshi0603** [comment]
actually losing my god damned mind LMFAO trying to do #30 
seems to work just fine but you run into a weird bug sometimes when bloating the register function

**Comment by xshi0603** [comment]
fixes #30 

we now use user instead of userInfo for navbar conditionals now, this should work fine but lmk if there are any bugs

**Comment by xshi0603** [comment]
fixes #36

**Comment by xshi0603** [comment]
plan on running prettier on our files later on, our indentation is so weird atm but wanted to save for a diff PR

File: client/src/pages/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const Profile = () => {
+  return (
+    <Container>
+      <div >
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+      </div>
```
Comment (sdhani) [inline client/src/pages/Profile.jsx:13]: I think we can replace this with css styling

**Review by sdhani — APPROVED** [review]

File: client/src/pages/Profile.jsx (hunk 1 above)
Comment (xshi0603) [reply client/src/pages/Profile.jsx:13]: i copied this off jessicas code LMFAO
 but yes prolly could be done via css

**Pull request merged**